phpcs.xml.dist

```diff
@@ -1,12 +1,11 @@
 <?xml version="1.0"?>
-<ruleset name="CakePHP Core">
- <rule ref="./vendor/cakephp/cakephp-codesniffer/CakePHP/ruleset.xml">
-    <!-- Exclude unwanted sniffs -->
-    <exclude name="Generic.Commenting.Todo.TaskFound"/> <!-- Excluded during 3.next development -->
- </rule>
+<ruleset name="CakePHP elastic-search">
+    <config name="installed_paths" value="../../cakephp/cakephp-codesniffer"/>
```

Comment:
Should this be `vendor/cakephp/cakephp-codesniffer`? Does PHPCS resolve this path based on CWD or where the phpcs binary is located?

Reply:
It can be, but I copied the format used everywhere. The relative path is always from the root codesniffer package directory.

Reply:
I'm not actually sure if installed path requires the absolute path if not relative.